.github/workflows/task-get-latest-tag.yml

```diff
@@ -18,6 +18,7 @@ on:
 
 jobs:
   get-tag: # Following best practices: https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28
+    name: Get Latest Release Tag for ${{ inputs.repo }} ${{ inputs.prefix }}
```

Comment:
Where do we use\see this name?

Reply:
`event-merge-to-queue` and `event-pull_request`